data/remnant/remnant outfits.txt

```diff
@@ -357,6 +360,7 @@ outfit "Aeon Cell"
 	"outfit space" -160
 	"energy generation" 21.9
 	"heat generation" 31.5
+	"ion protection" 0.45
```

Comment:
Heliarch ships have ion protection values ranging from 0.1 to 0.3, so values like these look way too high.

Instead of using ion protection, I'd suggest ONLY using ion resistance. Using ion resistance means that weak enough storms can be completely ignored while stronger storms only have a fraction of their ionization defended against.

And given that the main issue that ion storms cause with Remnant ships is the deactivation of their active cooling, maybe we only put the ion resistance on their cooling outfits. Whatever value that is used should also be small. Even a value of 0.05 ion resistance will cause a ship to ignore the ion damage on the very start and end of the weakest storms. How about we start with just that on the thermoelectric coolers? That would mean that 10 coolers (40 tons) would be enough to fight all the ionization of the weakest storms. 

If we want to do slightly more than that, keeping the ion resistance on the crystal capacitor should do.

Reply:
Regarding values, sure, I'll lower them. How would .3 for the largest one, then .2 for the middle, and .1 for the lowest look?

I had a few reasons for picking the outfits I did: 
1) We're dealing with ionization, which impacts energy production. So it makes sense that energy outfits that are getting hit are the most likely things that the Remnant would have focused on to solve the issue.
2) Remnant power generation is almost universally despised. The few people who use it (especially once they have access to Wanderer stuff), do so for RP reasons, not because they actually are better at anything. The Crystal Capacitor has awesome energy density... But everyone hates the power drain and heat it produces, so they avoid it like the plague. If we're going to put a buff on something, things that are generally poorly liked and underutilized makes more sense. (and the TEC already got a minor buff recently, so might as well share the love a bit.)
3) The three generators are larger items, so it'll require more deliberate choices for people to massively stack the resistance, rather than simply grabbing a bunch of fairly compact and already pretty good TECs.

As for the resistance vs protection, my thought process was:
- Generators produce power, and could be designed to withstand ionization.
- The crystal capacitor already has the lore that it bleeds energy - if it bleeds some ionization as well then that actually becomes a good thing, rather than purely a negative.

Reply:
I'm not sure why I wanted to go with thirds, but anyway. Current modification is .3, .1, and .033 protection respectively; with crystal capacitors unchanged at .1 resistance.

Reply:
Yeah, I'd still prefer the Remnant use resistance instead of protection.

Reply:
1. I don't see the Heliarch protection being cheapened by these outfits. Built in resistance is still massively superior to having to install sub-par generators. Especially since those ships with built-in protection are also some of the better ships in the game.
2. Warship level anti-ionization built into ships that, depending on the lore you go with, are either:
     - built to go up against enemies that raid them roughly once every century, and just minor raids at that.
     - built to go up against enemies that they "beat" once and expect to retaliate against them someday but they haven't seen for centuries, possibly millennia.
     Neither one of these really sounds like justification for a truly spectacular amount of ionic resistance. The urgency and import just isn't there. I'm not saying we should nerf the Heliarchs, I just don't see the need to have them held up as being ultra-special in this area. Especially not compared to faction(s) that deal with large amounts of ionization on a daily basis. 

They'll *still* be very special, this just opens up the possibility for ships to use sub-optimal generators to give other ships some protection too. Endless Sky is all about giving people choices, right? For a late game player who wants to avoid ionization, giving them the choice between high-powered Heliarch ships and other factions' regular ships sacrificing space to run sub-optimal generators... If I discount my fondness for the Remnant, I'd almost certainly pick the Heliarch.